### ISSUE TITLE
About TabItem.Header style customization problem

### ISSUE THREAD

**Issue by beefsteak** [spiegelp/MaterialDesignExtensions#126]
![屏幕截图 2021-02-02 161956](https://user-images.githubusercontent.com/14013918/106571785-8c552700-6572-11eb-8fb0-3bd15be9110c.png)
I want to give more custom permissions to the TabItem.Header, Like TabControlAssist helper class...
Ex:
TabControlAssist.TabItemHeaderMargin
TabControlAssist .TabItemHeaderHorizontalAlignment
TabControlAssist .TabItemHeaderVerticalAlignment

**Comment by beefsteak** [comment]
This gives the user the flexibility to add a lot of gameplay,
Ex:
![屏幕截图 2021-02-02 161956](https://user-images.githubusercontent.com/14013918/106572239-25843d80-6573-11eb-9b2a-eabe455a8d04.png)
Moving the mouse over the Header range displays the close button,But because the left and right are centered, I can't put the close button flexibly on the far right.

**Comment by Laritello** [comment]
I agree, run into related problem. Sometimes headers are just too big. I know with smaller margin and font size it would fit almost perfectly, but unfortunately I can't do it.

![image](https://user-images.githubusercontent.com/29901577/111700152-0ce19580-884a-11eb-80de-8b4e3d702922.png)
